dozer-sink-oracle/src/lib.rs

```diff
@@ -246,10 +245,10 @@ impl OracleSinkFactory {
         for field in &schema.fields {
             let name = &field.name;
             let col_type = match field.typ {
-                dozer_types::types::FieldType::UInt => "INTEGER",
-                dozer_types::types::FieldType::U128 => "INTEGER",
-                dozer_types::types::FieldType::Int => "INTEGER",
-                dozer_types::types::FieldType::I128 => "INTEGER",
+                dozer_types::types::FieldType::UInt => "NUMBER(20)",
+                dozer_types::types::FieldType::U128 => "NUMBER(39)",
```

Comment:
This will fail, because the maximum precision is 38. That basically means that we don't support 128 bit integers, because they won't fully fit in an Oracle number. We could try to support them anyway and error if they exceed the 39 decimal places precision.

Reply:
Right. Marking it as uinimplemented now. Later we will need to implement logic for it.